Project.toml

```
@@ -1,18 +1,23 @@
 name = "UnixTimes"
 uuid = "ab1a18e7-b408-4913-896c-624bb82ed7f4"
 authors = ["Christian Rorvik <christian.rorvik@gmail.com>"]
-version = "1.3.0"
+version = "1.3.1"
```

Not a bug fix, so I think we should bump minor for this.
```suggestion
version = "1.4.0"
```